helm-charts/charts/kube-starrocks/charts/starrocks/README.md

```diff
@@ -19,9 +19,9 @@
     $ helm repo update starrocks-community
     $ helm search repo starrocks-community
     NAME                                    CHART VERSION    APP VERSION  DESCRIPTION
-    starrocks-community/kube-starrocks      1.8.0            3.1-latest   kube-starrocks includes two subcharts, starrock...
+    starrocks-community/kube-starrocks      1.8.0            3.2-latest   kube-starrocks includes two subcharts, starrock...
```

Comment:
possibly keep this unchanged. the v1.8.0 chart is a static release with its snapshot back to the time. we just need to update the next chart release to the new `3.2-latest` version.

Reply:
done